### DISCUSSION TITLE
[BUG]  hiding top toolbar does not work

### DISCUSSION THREAD

**Discussion by BennyAlex** [KevinVandy/material-react-table#540] [General]
@KevinVandy 

https://codesandbox.io/s/trusting-mccarthy-81o1bz?file=/src/JS.jsx

![image](https://github.com/KevinVandy/material-react-table/assets/17402724/634867b8-9592-4912-9f20-c1004c8fb1db)

Top Toolbar is still shown even enableTopToolbar={false}

**Comment by KevinVandy** [comment]
When I open the link to your sandbox, it appears to be working fine

![Screenshot_20230518_180749_Chrome.jpg](https://github.com/KevinVandy/material-react-table/assets/28243511/ffb6dbd8-1e1b-47a2-b80c-ed5236162bae)

**Comment by KevinVandy** [comment]
Guessing the file just didn't get saved or there was a HMR issue

**Reply by BennyAlex** [reply]
Hm Yeah sorry I even testet in another Browser but now on mobile it's not there. Those boxes Always makes problems.

By the way, many links for the examples like the minimal and basic example to stackblitz as well as Code sanbox does not work.

On Stack Blitz I get a Memory RAM issue and on Code sanbox it throwes a Babel error.
